src/cio.ts

```diff
@@ -0,0 +1,10 @@
+import { TrackClient } from 'customerio-node';
+
+export const cio = new TrackClient(
+  process.env.CIO_SITE_ID,
+  process.env.CIO_API_KEY,
+);
```

Comment:
Just to make sure, will this work locally where we don't have credentials in env?

Reply:
i'll double check but it shouldn't break, probably will just throw an error if you try to actually run it, but not on init

Reply:
It is gonna fire when we create user locally right, and then worker will try to retry indefinitely. I think we had similar thing with sendgrid and then we just added NODE_ENV check to avoid running worker code where it is required in development.